libs/gltfio/src/MorphHelper.cpp

```diff
@@ -103,7 +103,7 @@ void MorphHelper::addPrimitive(cgltf_mesh const* mesh, int primitiveIndex, Entit
 
         auto& rcm = engine.getRenderableManager();
         rcm.setMorphTargetBufferAt(rcm.getInstance(entity), 0, primitiveIndex,
-                morphHelperPrim.targets, vertexBuffer->getVertexCount());
+                morphHelperPrim.targets);
```

Comment:
is that correct? `morphHelperPrim.targets` is the count of targets or count of vertices?

Reply:
Yes it's correct. We are set `MorphTargetBuffer` for `geometry` so it should be match `geometry`'s count. Also `MorphTargetBuffer`'s count(morph target count) should be match renderable's morph target count.

Reply:
oh you're right I misread the line. Sorry.

Reply:
No worry. Honestly it's very confused.